### PR TITLE
Payments | Create owner model with fixtures

### PR DESCRIPTION
**Pull request by lsoulworkerl**
- create Owner model with fixture
- create fixtures for PaymentService and PaymentCommission
- edit comission field in PaymentCommission (Float -> Decimal)
- add fields: is_paid and created_date in Invoice
- in ItemPurchase replaced (is_frozen, is_accepted) -> (status)
- fix functions return

### CODE REVIEW

**Comment by lsoulworkerl** [comment]
- split utils file
- add custom classmethod for deposit/withdraw
- create CommissionField

File: backend/payments/apps/base/utils/classmethod.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    operation_type: OperationType,
+):
+    if amount < 0:
+        raise ValueError('Should be posotive value')
+    with transaction.atomic():
+        obj = get_object_or_404(
+            django_model.objects.select_for_update(),
+            pk=pk,
+        )
+        balance = getattr(obj, django_field)
```
Comment (range-kun) [inline backend/payments/apps/base/utils/classmethod.py:28]: Need to handle case, when pased not existing field. 
Maybe we can check return type?

File: backend/payments/apps/base/utils/classmethod.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class OperationType(enum.Enum):
+    WITHDRAW = 'WITHDRAW'
+    DEPOSIT = 'DEPOSIT'
+
+
+def add_change_balance_method(
+    *,
+    django_model,
```
Comment (range-kun) [inline backend/payments/apps/base/utils/classmethod.py:15]: You can use DjangoModel type from base/utils.py.
Check parse_model_instance function inside that module for type annotation.

**Comment by lsoulworkerl** [comment]
Add handler in add_change_balance_method

**Review by kieled — APPROVED** [review]

**Pull request merged**